lingua_franca/res/text/uk-ua/normalize.json

```diff
@@ -69,6 +69,8 @@
     "тисяча": "1000",
     "тисяч": "1000"
   },
+  "symbols": [".", ",", ";", "_", "!", "?", "<", ">", "|", "(", ")", "=", "[",
+              "]", "{", "}", "»", "«", "*", "~", "^", "\""],
```

Comment:
@NeonDmitry I took the default list and removed "`"; I believe this would resolve the normalization errors you were seeing